packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -10,18 +10,21 @@ import { ClientSessionExpiredError, DataProcessingError, UsageError } from "@flu
 import { IRequestHeader } from "@fluidframework/core-interfaces";
 import {
     cloneGCData,
-    concatGarbageCollectionStates,
     concatGarbageCollectionData,
+    getGCDataFromSnapshot,
     IGCResult,
     runGarbageCollection,
     unpackChildNodesGCDetails,
 } from "@fluidframework/garbage-collector";
 import { ISnapshotTree, SummaryType } from "@fluidframework/protocol-definitions";
 import {
-    gcBlobKey,
+    gcTreeKey,
+    gcBlobPrefix,
+    gcTombstoneBlobKey,
     IGarbageCollectionData,
-    IGarbageCollectionState,
     IGarbageCollectionDetailsBase,
+    IGarbageCollectionSnapshotData,
+    IGarbageCollectionState,
```

Comment:
Nit: Looks like you deleted this just to reimport it

Reply:
I believe it just got alphabetically sorted.